### ISSUE TITLE
ns-train: error: unrecognized arguments: --trainer.load-dir

### ISSUE THREAD

**Issue by dapao1988** [autonomousvision/sdfstudio#124]
**Describe the bug**
I cannot "Resume from checkpoint / visualize existing run"

**To Reproduce**
1. This is the cmd about my first time to train:
ns-train neus-facto --pipeline.model.sdf-field.inside-outside False --vis viewer --experiment-name neus-facto-dtu65 sdfstudio-data --data data/sdfstudio-demo-data/dtu-scan65

2. When I found the result is not good, I decide to resume from that checkpoint on, But got the title error:
```bash
ns-train neus-facto --pipeline.model.sdf-field.inside-outside False --vis viewer --experiment-name neus-facto-dtu65 sdfstudio-data --data data/sdfstudio-demo-data/dtu-scan65 --trainer.load-dir=/dat
a2/cannon/work/aiar/sdfstudio/outputs/neus-facto-dtu65/neus-facto/2023-06-06_162348/sdfstudio_model

ns-train: error: unrecognized arguments: --trainer.load-dir {/data2/cannon/work/aiar/sdfstudio/outputs/neus-facto-dtu65/neus-facto/2023-06-06_162348/sdfstudio_models}
```

**Expected behavior**
I can recognize '--trainer.load-dir' arguments
becaurse I can see  'ns-train neus-facto --help', the command is :
<img width="465" alt="image" src="https://github.com/autonomousvision/sdfstudio/assets/5895953/f17d63ae-bb8c-4ebb-a4b9-e0191e488a81">

**Screenshots**
![image](https://github.com/autonomousvision/sdfstudio/assets/5895953/97d377c9-373f-44a2-b3b9-df0bc501923f)

**Additional context**
name: sdfstudio
channels:
  - defaults
dependencies:
  - _libgcc_mutex=0.1=main
  - _openmp_mutex=5.1=1_gnu
  - ca-certificates=2023.01.10=h06a4308_0
  - ld_impl_linux-64=2.38=h1181459_1
  - libffi=3.4.4=h6a678d5_0
  - libgcc-ng=11.2.0=h1234567_1
  - libgomp=11.2.0=h1234567_1
  - libstdcxx-ng=11.2.0=h1234567_1
  - ncurses=6.4=h6a678d5_0
  - openssl=1.1.1t=h7f8727e_0
  - python=3.8.16=h7a1cb2a_3
  - readline=8.2=h5eee18b_0
  - setuptools=67.8.0=py38h06a4308_0
  - sqlite=3.41.2=h5eee18b_0
  - tk=8.6.12=h1ccaba5_0
  - wheel=0.38.4=py38h06a4308_0
  - xz=5.4.2=h5eee18b_0
  - zlib=1.2.13=h5eee18b_0
  - pip:
    - absl-py==1.4.0
    - addict==2.4.0
    - anyio==3.7.0
    - appdirs==1.4.4
    - argon2-cffi==21.3.0
    - argon2-cffi-bindings==21.2.0
    - asttokens==2.2.1
    - attrs==23.1.0
    - av==9.2.0
    - babel==2.12.1
    - backcall==0.2.0
    - beautifulsoup4==4.12.2
    - bidict==0.22.1
    - bleach==6.0.0
    - cachetools==5.3.1
    - certifi==2023.5.7
    - cffi==1.15.1
    - charset-normalizer==3.1.0
    - click==8.1.3
    - comm==0.1.3
    - commonmark==0.9.1
    - configargparse==1.5.3
    - cryptography==38.0.0
    - cycler==0.11.0
    - dash==2.10.2
    - dash-core-components==2.0.0
    - dash-html-components==2.0.0
    - dash-table==5.0.0
    - debugpy==1.6.7
    - decorator==5.1.1
    - defusedxml==0.7.1
    - descartes==1.1.0
    - docker-pycreds==0.4.0
    - docstring-parser==0.14.1
    - exceptiongroup==1.1.1
    - executing==1.2.0
    - fastjsonschema==2.17.1
    - filelock==3.12.0
    - fire==0.5.0
    - flask==2.2.5
    - fonttools==4.39.4
    - frozendict==2.3.8
    - functorch==0.2.1
    - gdown==4.5.1
    - gitdb==4.0.10
    - gitpython==3.1.31
    - google-auth==2.19.1
    - google-auth-oauthlib==0.4.6
    - grpcio==1.54.2
    - h5py==3.8.0
    - idna==3.4
    - imageio==2.21.1
    - importlib-metadata==6.6.0
    - importlib-resources==5.12.0
    - ipykernel==6.23.1
    - ipython==8.12.2
    - ipython-genutils==0.2.0
    - ipywidgets==8.0.6
    - itsdangerous==2.1.2
    - jedi==0.18.2
    - jinja2==3.1.2
    - joblib==1.2.0
    - json5==0.9.14
    - jsonschema==4.17.3
    - jupyter==1.0.0
    - jupyter-client==8.2.0
    - jupyter-console==6.6.3
    - jupyter-core==5.3.0
    - jupyter-server==1.24.0
    - jupyterlab==3.3.4
    - jupyterlab-pygments==0.2.2
    - jupyterlab-server==2.22.1
    - jupyterlab-widgets==3.0.7
    - kiwisolver==1.4.4
    - lpips==0.1.4
    - markdown==3.4.3
    - markupsafe==2.1.2
    - matplotlib==3.5.3
    - matplotlib-inline==0.1.6
    - mediapy==1.1.0
    - mistune==2.0.5
    - msgpack==1.0.4
    - msgpack-numpy==0.4.8
    - nbclassic==0.5.6
    - nbclient==0.8.0
    - nbconvert==7.4.0
    - nbformat==5.7.0
    - nerfacc==0.3.5
    - nerfstudio==0.1.12
    - nest-asyncio==1.5.6
    - networkx==3.1
    - ninja==1.10.2.3
    - notebook==6.5.4
    - notebook-shim==0.2.3
    - numpy==1.24.3
    - nuscenes-devkit==1.1.9
    - oauthlib==3.2.2
    - open3d==0.17.0
    - opencv-python==4.6.0.66
    - packaging==23.1
    - pandas==2.0.2
    - pandocfilters==1.5.0
    - parso==0.8.3
    - pathtools==0.1.2
    - pexpect==4.8.0
    - pickleshare==0.7.5
    - pillow==9.5.0
    - pip==23.1.2
    - pkgutil-resolve-name==1.3.10
    - platformdirs==3.5.1
    - plotly==5.7.0
    - prometheus-client==0.17.0
    - prompt-toolkit==3.0.38
    - protobuf==3.20.0
    - psutil==5.9.5
    - ptyprocess==0.7.0
    - pure-eval==0.2.2
    - pyasn1==0.5.0
    - pyasn1-modules==0.3.0
    - pycocotools==2.0.6
    - pycparser==2.21
    - pygments==2.15.1
    - pymeshlab==2022.2.post2
    - pyngrok==5.1.0
    - pyparsing==3.0.9
    - pyquaternion==0.9.9
    - pyrsistent==0.19.3
    - pysocks==1.7.1
    - python-dateutil==2.8.2
    - python-engineio==4.4.1
    - python-socketio==5.7.1
    - pytz==2023.3
    - pywavelets==1.4.1
    - pyyaml==6.0
    - pyzmq==25.1.0
    - qtconsole==5.4.3
    - qtpy==2.3.1
    - requests==2.31.0
    - requests-oauthlib==1.3.1
    - rich==12.5.1
    - rsa==4.9
    - scikit-image==0.19.3
    - scikit-learn==1.2.2
    - scipy==1.10.1
    - send2trash==1.8.2
    - sentry-sdk==1.24.0
    - setproctitle==1.3.2
    - shapely==2.0.1
    - shtab==1.6.1
    - six==1.16.0
    - smmap==5.0.0
    - sniffio==1.3.0
    - soupsieve==2.4.1
    - stack-data==0.6.2
    - tenacity==8.2.2
    - tensorboard==2.9.0
    - tensorboard-data-server==0.6.1
    - tensorboard-plugin-wit==1.8.1
    - termcolor==2.3.0
    - terminado==0.17.1
    - threadpoolctl==3.1.0
    - tifffile==2023.4.12
    - tinycss2==1.2.1
    - tinycudann==1.7
    - torch==1.12.1+cu113
    - torch-fidelity==0.3.0
    - torchmetrics==0.11.4
    - torchtyping==0.1.4
    - torchvision==0.13.1+cu113
    - tornado==6.3.2
    - tqdm==4.65.0
    - traitlets==5.9.0
    - trimesh==3.21.7
    - typeguard==4.0.0
    - typing-extensions==4.6.2
    - tyro==0.5.3
    - tzdata==2023.3
    - u-msgpack-python==2.8.0
    - urllib3==1.26.16
    - wandb==0.15.3
    - wcwidth==0.2.6
    - webencodings==0.5.1
    - websocket-client==1.5.2
    - werkzeug==2.2.3
    - widgetsnbextension==4.0.7
    - xatlas==0.0.7
    - zipp==3.15.0
prefix: /home/xxx/anaconda3/envs/sdfstudio

**Comment by niujinshuchong** [comment]
Yes, order matters. Thanks for reporting it. Will update the docs soon.

**Comment by niujinshuchong** [comment]
Docs is updated. Thanks again.
